tools/social-twitter/src/tweet/get_recent_search_tweets.rs

```diff
@@ -0,0 +1,845 @@
+//! # xyz.taluslabs.social.twitter.get-recent-search-tweets@1
+//!
+//! Standard Nexus Tool that retrieves tweets from the recent search API.
+
+use {
+    crate::{
+        error::TwitterErrorKind,
+        tweet::models::{
+            ExpansionField, Includes, MediaField, Meta, PlaceField, PollField, SortOrder, Tweet,
+            TweetField, TweetsResponse, UserField,
```

Comment:
`cargo +nightly fmt` 

you can enable this automatically in VSCode by adding:


`"rust-analyzer.rustfmt.extraArgs": ["+nightly"],`

Reply:
No worries, cheers!